### PR TITLE
Alchemy tutorials [#1751 and #1752]

### PR DESCRIPTION
**Pull request by ryancreatescopy**
## Description

Added Alchemy tutorials.

## Related Issue

#1751 
#1752

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
One image (a gif) in this PR isn't working due to the following bug/issue: #1845

**Comment by wackerow** [comment]
See PR #1851 for GIF fix

File: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+![Mempool watcher screenshot](./mempool.png)
+
+From there you can view your transaction on Etherscan by clicking on the icon circled in red!
+
+**Yippieeee! You just sent your first Ethereum transaction using Alchemy 🎉**
+
+_For feedback and suggestions about this guide, please message Elan on Alchemy’s [Discord](https://discord.gg/A39JVCM)!_
+
+_Originally published at_ [](https://docs.alchemyapi.io/tutorials/sending-transactions-using-web3-and-alchemy) [https://docs.alchemyapi.io/tutorials/sending-transactions-using-web3-and-alchemy](https://docs.alchemyapi.io/tutorials/sending-transactions-using-web3-and-alchemy)
```
Comment (samajammin) [inline src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:151]:
```suggestion
_Originally published at [https://docs.alchemyapi.io/tutorials/sending-transactions-using-web3-and-alchemy](https://docs.alchemyapi.io/tutorials/sending-transactions-using-web3-and-alchemy)_
```

File: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+- [`eth_sendTransaction`](https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#eth-sendtransaction) is used for sending _unsigned_ transactions, which means the node you are sending to must manage your private key so it can sign the transaction before broadcasting it to the chain. Since Alchemy doesn't hold user's private keys, we do not support this method.
+- [`eth_sendRawTransaction`](https://docs.alchemyapi.io/documentation/alchemy-api-reference/json-rpc#eth_sendrawtransaction) is used to broadcast transactions that have already been signed. This means you first have to use [`signTransaction(tx, private_key)`](https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#signtransaction), then pass in the result into `eth_sendRawTransaction`.
+
+When using web3, `eth_sendRawTransaction` is accessed by calling the function [web3.eth.sendSignedTransaction](https://web3js.readthedocs.io/en/v1.2.0/web3-eth.html#sendsignedtransaction).
+
+This is what we will be using in our tutorial.
+
+### 6\. What is the web3 library?
+
+- Web3 is a wrapper library around the standard JSON-RPC calls that is quite common to use in Ethereum development.
```
Comment (samajammin) [inline src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:54]:
```suggestion
- Web3.js is a wrapper library around the standard JSON-RPC calls that is quite common to use in Ethereum development.
```

File: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- There are many web3 libraries for different languages. In this tutorial we’ll be using Alchemy web3 which is written in JavaScript. You can check out other options [here](https://docs.alchemyapi.io/guides/getting-started#other-web3-libraries).
```
Comment (samajammin) [inline src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:55]:
```suggestion
- There are many web3 libraries for different languages. In this tutorial we’ll be using [Alchemy Web3](https://docs.alchemyapi.io/documentation/alchemy-web3) which is written in JavaScript. You can check out other options [here](https://docs.alchemyapi.io/guides/getting-started#other-web3-libraries).
```

File: src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+For manual requests, we recommend interacting with the `JSON-RPC` via `POST` requests. Simply pass in the `Content-Type: application/json` header and your query as the `POST` body with the following fields:
+
+- `jsonrpc`: The JSON-RPC version—currently, only `2.0` is supported.
+- `method`: The ETH API method. [See API reference.](https://docs.alchemyapi.io/documentation/alchemy-api-reference/json-rpc)
+- `params`: A list of parameters to pass to the method.
+- `id`: The ID of your request. Will be returned by the response so you can keep track of which request a response belongs to.
+
+Here is an example you can run from the command line to retrieve the current gas price:
+
+```
```
Comment (samajammin) [inline src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:56]:
```suggestion
```bash
```

File: src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+-X POST \
+-H "Content-Type: application/json" \
+-d '{"jsonrpc":"2.0","method":"eth_gasPrice","params":[],"id":73}'
+```
+
+**_NOTE:_** _Replace_ [_https://eth-mainnet.alchemyapi.io/v2/demo_](https://eth-mainnet.alchemyapi.io/jsonrpc/demo) _with your own API key_ [_https://eth-mainnet.alchemyapi.io/v2/your-api-key_](https://eth-mainnet.alchemyapi.io/jsonrpc/your-api-key)_._
+
+**Results:**
+
+```
```
Comment (samajammin) [inline src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:67]:
```suggestion
```json
```

File: src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+**With NPM:**
+
+```
+npm install @alch/alchemy-web3
+```
+
+To interact with Alchemy’s node infrastructure, run in NodeJS or add this to a JavaScript file:
+
+```
```
Comment (samajammin) [inline src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:95]:
```suggestion
```js
```

File: src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+npm install @alch/alchemy-web3
+```
+
+**‌3. Create a file named** `index.js` **and add the following contents:**
+
+> You should ultimately replace `demo` with your Alchemy HTTP API key.
+
+```
```
Comment (samajammin) [inline src/content/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:119]:
```suggestion
```js
```

**Review by samajammin — CHANGES_REQUESTED** [review]
Made a few edits.

**Review by samajammin — APPROVED** [review]

**Pull request merged**